crates/gitbutler-tauri/src/main.rs

```diff
@@ -56,6 +56,16 @@ fn main() -> anyhow::Result<()> {
     ) {
         tauri_context.config_mut().app.security.csp = updated_csp;
     };
+
+    if let Some(project_to_open) =
+        std::env::var_os("GITBUTLER_PROJECT_DIR").map(std::path::PathBuf::from)
+    {
+        bail!(
```

Comment:
@estib-vega Do you know how to get the (probably) project-id to the frontend so it opens that instead?
Something I could imagine is that the frontend figures out what the project to open is by asking the backend, instead of storing that itself.
Then this information could be wired up.

For now this can probably be done by project-id, even though in future I'd hope we can get rid of it.